src/graph_spectrum_plot.js

```diff
@@ -1470,13 +1520,26 @@ GraphSpectrumPlot._drawMousePosition = function (
       );
     }
 
+    if (this._spectrumType === SPECTRUM_TYPE.POWER_SPECTRAL_DENSITY) {
+      const psdLabel = Math.round(this.getPSDbyFreq(mouseFrequency)).toString() + "dBm/Hz";
+      this._drawAxisLabel(
+            canvasCtx,
+            psdLabel,
+            mouseX - 30,
+            mouseY - 4,
+            "left",
```

Comment:
Indentation is 2 now ?

```suggestion
        canvasCtx,
        psdLabel,
        mouseX - 30,
        mouseY - 4,
        "left",
```